sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Microsoft.Azure.WebPubSub.AspNetCore.csproj

```diff
@@ -14,9 +14,9 @@
 
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="System.Text.Json" />
-    <PackageReference Include="System.Text.Encodings.Web" />
-    <PackageReference Include="System.Memory.Data" />
+    <PackageReference Include="System.Text.Json" VersionOverride="6.0.11" />
```

Comment:
This package targets netcoreapp3.1. BCL 8.x packages aren't tested with netcoreapp3.1 and aren't guaranteed to work with that target.